client/gutenberg/extensions/presets/jetpack/index.json

```diff
@@ -1,3 +1,4 @@
 [
-  "markdown"
+  "markdown",
+  "publicize"
```

Comment:
Suppose you'll drop this out before merging?

Reply:
Can do. Although I thought we can merge it - that way we won't have to add it to every Publicize  PR.

Reply:
That commit has already been merged with #28187.